exec/cnex/lib/os_posix.c

```diff
@@ -0,0 +1,192 @@
+// The following is needed for kill() on Posix platforms.
+#define _POSIX_C_SOURCE 200809L
+
+#include <assert.h>
+#include <errno.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <sys/param.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+
+#include "cell.h"
+#include "enums.h"
+#include "exec.h"
+#include "number.h"
+#include "object.h"
+#include "stack.h"
+
+
+
+static Array *g_children;
+
+typedef struct tagTProcessHandle {
+    pid_t proc;
```

Comment:
Were you going to change this from `proc` back to `pid`?

Reply:
Sorry, that got missed.  Fixed now.